scripts/formats.sh

```diff
@@ -1,4 +1,4 @@
 stage()
 {
-  tput setaf 4 && tput bold && echo "$1" && tput sgr0
+  ([ -n "$(tput setaf 1)" ] && { tput setaf 4 && tput bold && echo "$1" && tput sgr0; }) || echo "$1"
```

Comment:
```suggestion
  ([ -n "$(tput setaf 1 2<&-)" ] && { tput setaf 4 && tput bold && echo "$1" && tput sgr0; }) || echo "$1"
```
This will close the stderr descriptor from `tput` so that the logs won't contain `tput` whining about `TERM` variable:
<img width="286" alt="image" src="https://github.com/ostis-ai/sc-web/assets/43214067/be5e5c2e-5e2c-46b5-b6a7-55bab47217ee">